sahi/utils/file.py

```diff
@@ -39,7 +39,7 @@ def save_json(data, save_path):
 
     # export as json
     with open(save_path, "w", encoding="utf-8") as outfile:
-        json.dump(data, outfile, separators=(",", ":"), cls=NumpyEncoder)
+        json.dump(data, outfile, separators=(",", ":"), cls=NumpyEncoder,ensure_ascii=False)
```

Comment:
This may not be always wanted, I think it'd be more proper if we add this, ass an argument to `save_json()` function.

Reply:
@fcakyon I'm not sure if in sahi we could potentially use `ensure_ascii=False` anywhere, but making it a function argument and let user decide is more generic and better option IMO (to be on safe-side). There's a [discussion in SO](https://stackoverflow.com/q/40412714) about the usage.

Reply:
yes it seems in most cases `ensure_ascii=False` cause trouble.